lib/embulk/input/jira_api/client.rb

```diff
@@ -67,7 +67,9 @@ def timeout_and_retry(wait, retry_times = DEFAULT_SEARCH_RETRY_TIMES, &block)
                 # a.k.a. HTTP 503
                 raise title
               when "Unauthorized (401)"
-                raise Embulk::ConfigError.new("JIRA returns error: #{title}")
+                puts "JIRA returns error: #{title}"
```

Comment:
`Embulk.logger.warn` is better than `puts` for logging.

Reply:
Thanks @uu59. I will update